### DISCUSSION TITLE
Open File Picker by default when simply doing "hx"

### DISCUSSION THREAD

**Discussion by adriangalilea** [helix-editor/helix#11573] [General]
Similar to https://github.com/helix-editor/helix/issues/51

But I would like to happen this by default when I type `hx` instead of having to do `hx .` 95% of the time I do hx is to navigate with fzf and find the thing I want to edit rather than creating a new file.

**Comment by jarjk** [comment]
For me, it's just the opposite.

What about making some function in `$SHELL`, to check the num of args, and if none is received, use `hx .` else `hx $FILE`?

**Comment by adriangalilea** [comment]
> For me, it's just the opposite.
> 
> What about making some function in `$SHELL`, to check the num of args, and if none is received, use `hx .` else `hx $FILE`?

What do you mean by the opposite, I want that when I type `hx` the file picker is open, and when I type `hx $FILE` then that file opens without the file picker, we want the same, no?

**Comment by jarjk** [comment]
Well, I meant the opposite, as 95% of the time, I use `hx $file`, and in the remaining 5% I either open a directory, other than `.`, or want an empty buffer, meaning I prefer the functionality as is.

Although it's not completely the same as you suggest, I'd love a feature (though not particularly need it) that does something like this neovim plugin: [alpha-nvim](https://github.com/goolord/alpha-nvim)

**Comment by adriangalilea** [comment]
> Well, I meant the opposite, as 95% of the time, I use `hx $file`, and in the remaining 5% I either open a directory, other than `.`, or want an empty buffer, meaning I prefer the functionality as is.
> 
> Although it's not completely the same as you suggest, I'd love a feature (though not particularly need it) that does something like this neovim plugin: [alpha-nvim](https://github.com/goolord/alpha-nvim)

I see, so it's best to solve as yet another flag in the config, but I'd say this is fairly low priority so perhaps after steel plugins to no add more weight

**Comment by Axlefublr** [comment]
an option in the config could work, but this feels like a plugin angle
